moveit_core/collision_distance_field/src/collision_env_distance_field.cpp

```diff
@@ -1088,7 +1088,7 @@ PosedBodySphereDecompositionPtr
 CollisionEnvDistanceField::getPosedLinkBodySphereDecomposition(const moveit::core::LinkModel* ls, unsigned int ind) const
 {
   PosedBodySphereDecompositionPtr ret;
-  ret.reset(new PosedBodySphereDecomposition(link_body_decomposition_vector_[ind]));
+  ret = std::make_shared<PosedBodySphereDecomposition>(link_body_decomposition_vector_[ind]);
```

Comment:
Small nit can you change vector operator[] to at()?  Feel free to ignore this and we can do it in a separate change.

Reply:
Good catch. I will change it.